packages/edit-site/src/components/header/index.js

```diff
@@ -25,39 +22,38 @@ import SaveButton from '../save-button';
 import UndoButton from './undo-redo/undo';
 import RedoButton from './undo-redo/redo';
 import DocumentActions from './document-actions';
-import NavigationToggle from './navigation-toggle';
 
 export default function Header( { openEntitiesSavedStates } ) {
-	const {
-		deviceType,
-		hasFixedToolbar,
-		template,
-		isNavigationOpen,
-		isInserterOpen,
-	} = useSelect( ( select ) => {
-		const {
-			__experimentalGetPreviewDeviceType,
-			isFeatureActive,
-			getTemplateId,
-			isNavigationOpened,
-			isInserterOpened,
-		} = select( 'core/edit-site' );
-		const { getEntityRecord } = select( 'core' );
+	const { deviceType, hasFixedToolbar, template, isInserterOpen } = useSelect(
+		( select ) => {
+			const {
+				__experimentalGetPreviewDeviceType,
+				isFeatureActive,
+				getTemplateId,
+				isNavigationOpened,
```

Comment:
Looks like we can remove this since we aren't using it any more

Reply:
Good catch